IPlug/IPlugBase.h

```diff
@@ -119,9 +137,16 @@ class IPlugBase
   // Useful stuff for your plugin class or an outsider to call,
   // most of which is implemented by the API class.
 
+  /** @brief Get sample rate (in Hz)
+   * @return Sample rate (in Hz). Defaults to 44100.0
+   */
   double GetSampleRate() const { return mSampleRate; }
   int GetBlockSize() const { return mBlockSize; }
+  /** @return Plugin latency (in samples)
+   * @todo Please check this and remove this note once done
+   */
   int GetLatency() const { return mLatency; }
+  /** @return \c True if the plugin is currently bypassed */
   bool GetIsBypassed() const { return mIsBypassed; }
```

Comment:
This is really ugly, can we GetIsBypassed() -> IsBypassed() or GetIsBypassed -> GetBypassed() for IPlug2?

Reply:
Yes I’ve been thinking about prefixing every getter with get

Reply:
Is- works for telling you it's a bool instantly, but that's your call, either is fine, just need to stick with just one style

Reply:
What do you think about things like NParams() NPresets()?

Reply:
I think getNumParams() would be way more descriptive

Reply:
could you create a separate issue for suggestion such as this? to help keep track